src/EPR.Payment.Facade/Controllers/RegistrationFees/ProducersFeesController.cs

```diff
@@ -98,7 +98,7 @@ public async Task<IActionResult> CalculateFeesAsync([FromBody] ProducerFeesReque
             }
         }
 
-        [HttpGet]
+        [HttpGet("resubmission")]
```

Comment:
If you are changing the route in this ticket as per the meeting with Ramya, then change it properly. The route should be 
........./v1/producer/registration-fee

Reply:
it covered with another ticket